pkg/reconciler/namespace/namespace.go

```diff
@@ -141,6 +150,13 @@ func (r *Reconciler) reconcile(ctx context.Context, ns *corev1.Namespace) error
 // reconcileServiceAccountAndRoleBinding reconciles the service account and role binding for
 // Namespace 'ns'.
 func (r *Reconciler) reconcileServiceAccountAndRoleBindings(ctx context.Context, ns *corev1.Namespace, saName, rbName, clusterRoleName, configClusterRoleName string) error {
+
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		r.Recorder.Event(ns, corev1.EventTypeNormal, envConfigProcessed,
```

Comment:
we usually don't record an event when the env is processed. Maybe we should. 

Also I suggest to move it into `controller.go` so that it's done only once. Here an example: https://github.com/knative/eventing/blob/master/pkg/reconciler/apiserversource/controller.go#L74

Reply:
Just moved this over to controller.go. For now, let's keep do this in the same way we treat other env processing.